src/common/PersonTile/styled.js

```diff
@@ -69,4 +70,52 @@ export const Name = styled.div`
   @media (max-width: ${({ theme }) => theme.breakpoint.mobileMin}px) {
     font-size: 14px;
   }
+  ${({ list }) =>
+    list &&
+    css`
+      -webkit-line-clamp: 2;
+      max-height: 2.6em;
+    `}
 `;
+
+
+export const Person = styled.article`
+  padding: 16px;
+  background-color: ${({ theme }) => theme.color.white};
+  box-shadow: ${({ theme }) => theme.shadow};
+  border-radius: 5px;
+  text-align: center;
+  height: 100%;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.medium}) {
+    padding: 8px;
+  }
+`;
+export const Role = styled.span`
+  color: ${({ theme }) => theme.color.waterloo};
```

Comment:
you missed the "s" :) please replace with : color: ${({ theme }) => theme.colors.waterloo}; 👍

Reply:
Thanks! My keyboard has been failing me lately ;c